docs/installation.rst

```diff
@@ -50,12 +50,19 @@ Execute the following command to install latest DALI for specified CUDA version:
 
    pip install --extra-index-url https://developer.download.nvidia.com/compute/redist nvidia-dali-cuda100
 
-* for CUDA 11:
+* for CUDA 11 and CUDA 11.1:
+
+  .. note::
+    CUDA 11.1 builds uses CUDA toolkit extended compatibility. It is built with the latest CUDA 11.x
+    toolkit while it can run on CUDA 11.0 capable drivers 450.80 or later. Using the newest driver may enable
+    additional functionalities.
```

Comment:
It would be nice to link to some docs about extended compatibility. Googling that term brings nothing as of now.

Reply:
I will add it when it is available